app/src/include/firebase/app.h

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-#ifndef FIREBASE_APP_CLIENT_CPP_SRC_INCLUDE_FIREBASE_APP_H_
-#define FIREBASE_APP_CLIENT_CPP_SRC_INCLUDE_FIREBASE_APP_H_
+#ifndef FIREBASE_APP_SRC_INCLUDE_FIREBASE_APP_H_
+#define FIREBASE_APP_SRC_INCLUDE_FIREBASE_APP_H_
 
```

Comment:
📝 &nbsp;__Documentation issue:__ `warning: argument 'confirg' of command @param is not found in the argument list of firebase::AppOptions::LoadFromJsonConfig(const char *config, AppOptions *options=nullptr)`<hidden value="hidden-doxygen-comment-tag"></hidden>